### ISSUE TITLE
Browse by author shows no results when ORCID authority enabled

### ISSUE THREAD

**Issue by uofmsean** [DSpace/dspace-angular#1856]
**Describe the bug**
DSpace 7.3
Browse by author does not show results when ORCID authority is enabled and indexing is configured with:
```
webui.browse.index.2 = author:metadataAuthority:dc.contributor.author:authority
```

The bug is due to a missing "authority" value in the BrowseByDataType enum, and also that the type is not associated with a corresponding component (such as, for example, a new browse-by-authority-page.component.ts).

Note that the REST API endpoint /server/api/discover/browses/author/entries does return results when queried directly so the issue is not with missing data.

**To Reproduce**
1. Enable ORCID authority for dc.contributor.author
2. Update Solr cores as necessary (index-authority and index-discovery -b)
3. Select By Author from the All of DSpace menu
4. No results will show and there is no error message because there is no matching component to render

**Expected behavior**
Authority controlled authors should be listed in any page that displays a "browse by author" component., either for all of DSpace or contextually for a collection or community.

**Related work**
First mentioned [in this comment](https://github.com/DSpace/dspace-angular/issues/573#issuecomment-1249838894) while reviewing issue #573

**Comment by uofmsean** [comment]
This looks like it has been fixed with DSpace 7.4. I think BrowseByDataType.Metadata gets used as the default and authority-controlled author names are now showing. This was after a full search rebuild (index-discovery -b) so it was fresh data.

**Comment by tdonohue** [comment]
Thanks @uofmsean !  Since you were the original reporter, I'll close this as fixed in 7.4. If we find further issues later, we can either reopen or create a new ticket.
